src/renderer/atlas/AtlasEngine.r.cpp

```diff
@@ -53,15 +62,15 @@ using namespace Microsoft::Console::Render;
 [[nodiscard]] HRESULT AtlasEngine::Present() noexcept
 try
 {
-    _adjustAtlasSize();
-    _processGlyphQueue();
-
-    if (WI_IsFlagSet(_r.invalidations, RenderInvalidations::Cursor))
+    if (_r.d2dMode)
     {
-        _drawCursor();
-        WI_ClearFlag(_r.invalidations, RenderInvalidations::Cursor);
+        _d2dPresent();

```

Comment:
should we not return the return value of `_d2dPresent`? so we can detect failure?

Reply:
`_d2dPresent` throws on error. I don't really use status codes in this class.